src/MarkdownTextInput.web.tsx

```diff
@@ -68,6 +69,11 @@ type Selection = {
   end: number;
 };
 
+type Dimensions = {
+  height: number | null;
+  width: number | null;
+};
```

Comment:
This type allows `height` to be set and `width` to be null, is this desired? How about changing this to
```suggestion
type Dimensions = {
  width: number;
  height: number;
};
```
and using `Dimensions | null` in the code below?